cmake/DaemonGame.cmake

```diff
@@ -77,6 +77,8 @@ function(GAMEMODULE)
             endforeach(inherited_option)
```

Comment:
The variable `vm` defined a bit above is PNaCl-specific (and redundant with the newly added NACL_VMS_PROJECT).

Reply:
Removed.